policyengine_us/parameters/gov/states/co/tax/income/credits/ctc/ctc_matched_federal_credit.yaml

```diff
@@ -0,0 +1,12 @@
+description: Whether the Colorado child tax credit matches the federal child tax credit.
```

Comment:
```suggestion
description: Colorado matches a fraction of the federal child tax credit when this is true; otherwise, Colorado provides an amount based on income.
```